Resources/Prototypes/Entities/Mobs/Species/base.yml

```
@@ -458,6 +458,7 @@
   - type: LayingDown # WD EDIT
   - type: Targeting # Shitmed Change
   - type: CanEscapeInventory # GoobStation - Escape being picked up by players
+  - type: ExaminableCharacter # WWDP
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 2

(indentation)